idaes/models/properties/modular_properties/phase_equil/tests/test_smooth_VLE_2.py

```diff
@@ -339,3 +343,162 @@ def test_calculate_ceos_derivative_slacks(frame):
 
     assert value(gp["Vap"]) == pytest.approx(EPS_INIT, abs=1e-8)
     assert value(gn["Vap"]) == pytest.approx(EPS_INIT, abs=1e-8)
+
+
+class TestWithNonCondensable:
+    # TODO These tests could be fleshed out
```

Comment:
What would this TODO entail?

Reply:
Mostly adding in tests for the values of `gp`, `gn`, `_teq`, and other VLE variables both to make sure the model isn't changing and that the answer is reasonable. Right now, I'm testing that 1) the model initialization doesn't end up with an error and 2) bubble/dew points are not erroneously created.